app/page/template/_dist/app.htm

```diff
@@ -409,6 +409,7 @@
 <!-- View -->
 <script src="/script/ui/AvailabilityContextMenu.js"></script>
 <script src="/script/ui/ContextMenu.js"></script>
+<script src="/script/ui/OverlayedObserver.js"></script>
 <script src="/script/ui/ShortcutType.js"></script>
 <script src="/script/ui/Shortcut.js"></script>
 <script src="/script/ui/WindowHandler.js"></script>
```

Comment:
🙈 Looks like my ABC is not too good. Can you please change `WindowHandler` and `ViewportObserver` as well.